week1/lecture/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.1",
   "description": "HackYourFuture Node.js Week 1 - Lecture",
   "repository": "https://github.com/HackYourFuture/nodejs.git",
-  "main": "src/index.js",
+  "main": "Ivysproject/testingNode.js",
```

Comment:
Why was this changed?

Reply:
I had src/server.js by accident there and had to change it to index.js so as to get localhost to run

Reply:
I also had configuration from the example folder(Ivysproject) added by me in the week1 json file by mistake...thereby the modification to removing line 6 so that I wouldn´t get errors

Reply:
this chould point to this project not to another project it should work with src/index.js